bridge/engine/EngineMixer.cpp

```diff
@@ -4169,7 +4169,7 @@ void EngineMixer::onBarbellUserMediaMap(size_t barbellIdHash, const char* messag
         }
     }
 
-    memory::Map<size_t, BarbellMapItem, 8> audioSsrcs;
+    memory::Map<size_t, BarbellMapItem, 10> audioSsrcs;
```

Comment:
As we have 5 audioSsrc I was thinking that were possible 10 changes. But I see now that is wrong. I will revert it